webui/settings.py

```diff
@@ -128,6 +128,7 @@
 
 USE_I18N = True
 
+# DO NOT CHANGE THIS VALUE!
```

Comment:
maybe few words about WHY it is bad idea to change it ?

Reply:
Done.